pyproject.toml

```diff
@@ -34,3 +34,4 @@ build-backend = "poetry.core.masonry.api"
 yoeo-detect = "yoeo.detect:run"
 yoeo-train = "yoeo.train:run"
 yoeo-test = "yoeo.test:run"
+yoeo-download-weights = "scripts.download_weights:run"
```

Comment:
does this really work? Script seems like a non install directory, as it is not in the package.

Reply:
yes

Reply:
TDB lets test this with a clean install